java-spring/build.gradle

```diff
@@ -34,12 +37,12 @@ repositories {
 
 dependencies {
     implementation 'io.crate:crate-jdbc:2.6.0'
-    implementation 'commons-codec:commons-codec:1.14'
-    implementation 'org.springframework.boot:spring-boot-starter-web:2.2.6.RELEASE'
-    implementation 'org.springframework.boot:spring-boot-starter-data-jdbc:2.2.6.RELEASE'
-    implementation 'org.springframework.boot:spring-boot-starter-jdbc:2.2.6.RELEASE'
-    testImplementation 'org.springframework.boot:spring-boot-starter-test:2.2.6.RELEASE'
-    testImplementation 'junit:junit'
+    implementation 'commons-codec:commons-codec:1.15'
+    implementation 'org.springframework.boot:spring-boot-starter-web:2.2.13.RELEASE'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jdbc:2.2.13.RELEASE'
+    implementation 'org.springframework.boot:spring-boot-starter-jdbc:2.2.13.RELEASE'
+    testImplementation 'org.springframework.boot:spring-boot-starter-test:2.2.13.RELEASE'
+    testImplementation 'junit:junit:4.13.2'
```

Comment:
@mkleen: Unfortunately, we can't go beyond Spring Boot 2.2 here. With 2.3, the application will croak on startup with `Couldn't determine Dialect for "crate"`.

```
2023-01-18 15:29:45.510  INFO 74918 --- [           main] o.s.d.j.r.config.DialectResolver         : Couldn't determine Dialect for "crate"
2023-01-18 15:29:45.514  WARN 74918 --- [           main] ConfigServletWebServerApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jdbcConverter' defined in class path resource [org/springframework/boot/autoconfigure/data/jdbc/JdbcRepositoriesAutoConfiguration$SpringBootJdbcConfiguration.class]: Unsatisfied dependency expressed through method 'jdbcConverter' parameter 4
```